<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FitModelTabContent</class>
 <widget class="QWidget" name="FitModelTabContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Fit model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Optimize a model to your data.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <item row="4" column="0" colspan="3">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="selectDWI">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="selectedDWI">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Select DWI:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>(Select your protocol file, see tab &quot;Generate protocol file&quot;)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Select brain mask:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>(Select your preprocessed 4d diffusion weighted volume)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Select model:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="modelSelection"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>(Select your brain mask, see tab &quot;Generate brain mask&quot;)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_13">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Select output folder:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Select protocol file:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>(Defaults to &quot;output/&lt;mask_name&gt;&quot; in the DWI directory)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="selectOutputFolder">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="selectedOutputFolder">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>(Please select a model)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="selectMask">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="selectedMask">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="selectProtocol">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="selectedProtocol">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="additionalDataButton">
         <property name="text">
          <string>Additional data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="optimizationOptionsButton">
         <property name="text">
          <string>Optimization options</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="label_17">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>(Additional settings)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="runButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>line_2</zorder>
  <zorder>line</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <tabstops>
  <tabstop>selectDWI</tabstop>
  <tabstop>selectedDWI</tabstop>
  <tabstop>selectMask</tabstop>
  <tabstop>selectedMask</tabstop>
  <tabstop>selectProtocol</tabstop>
  <tabstop>selectedProtocol</tabstop>
  <tabstop>selectOutputFolder</tabstop>
  <tabstop>selectedOutputFolder</tabstop>
  <tabstop>modelSelection</tabstop>
  <tabstop>additionalDataButton</tabstop>
  <tabstop>optimizationOptionsButton</tabstop>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
